app/src/main/java/me/phh/treble/app/Misc.kt

```diff
@@ -44,10 +44,15 @@ object Misc: EntryStartup {
                 Log.d("PHH", "Setting signal level method to $value")
             }
             MiscSettings.fpsDivisor -> {
-                val value = sp.getString(key, "1")
```

Comment:
What

Reply:
Ugh I seriously don't know how I manage to do such things... Changed